### DISCUSSION TITLE
Tonic minimum dependency

### DISCUSSION THREAD

**Discussion by Cloud33** [hyperium/tonic#1313] [Q&A]
I am a beginner. I see that there are many options in tonic features, but I don't know how to configure the minimum dependency. I find that there are many web services related, such as axum, hyper
I just want to use super+warp to make a simple grpc server. What features should I add
My idea:
```rust
tonic = { version ="0.8.3", features = ["channel", "prost", "gzip"] }
```
correct?

**Comment by LucioFranco** [comment]
You need to disable the default features.

**Reply by Cloud33** [reply]
this?
```rust 
tonic = { version ="0.8.3", default-features = false, features = ["channel", "prost", "gzip"] }
```

**Reply by LucioFranco** [reply]
yes

**Reply by Cloud33** [reply]
thank you
